doc/protocol.md

```diff
@@ -77,8 +77,8 @@ open until the "init" message has been received.
 
 The following fields are defined:
 
- * "version": The version of the protocol. Currently zero, and unstable.
- * "capabilities": An array of capability strings
+ * "version": The version of the protocol. Currently 1, and stable.
```

Comment:
Should we have a link here to a wiki page regarding what "stable" actually means? i.e. compatibility